metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -142,7 +142,6 @@ public ObjectsImporter(
         this.taxonomyEventHandler = taxonomyEventHandler;
     }
 
-    @Transactional(rollbackFor = Exception.class)
```

Comment:
@tskir I am fine with making the trade-off  for performance. But, have you confirmed the original reasoning behind the transactional annotation (with the previous devs, perhaps)? Was that to prevent stale reads while the relevant Oracle table is being updated? If there are specific downsides to not having this as a transaction, could you please document it through comments/readme etc.,?

Reply:
Luckily I already know the original reasoning because I was there when the decision was made. The idea was that if we make the import transactional, it will always result in a clean (but possibly incomplete) state in the target database, as any study or analysis with exceptions during import would be cleanly rolled back. This is not important for the EVA use case anyway, since even a single unexpected exception in the import process means that it has ultimately failed. I've added the explanation to documentation: https://github.com/EBIvariation/metadata-ws/pull/86/commits/608ed7c7bf51ffab28ecd9637833f8cf5bf94477